policyengine_us/variables/gov/states/md/tax/income/credits/ctc/md_non_refundable_ctc.py

```diff
@@ -0,0 +1,14 @@
+from policyengine_us.model_api import *
+
+
+class md_non_refundbale_ctc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Maryland non-refundable Child Tax Credit"
+    definition_period = YEAR
+    unit = USD
+    documentation = "Maryland Child Tax Credit"
+    reference = "https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-751-effective-until-712026-tax-credit-for-qualified-child"
+    defined_for = StateCode.MD
+
```

Comment:
@MaxGhenis before 2023 the [law](https://law.justia.com/codes/maryland/2022/tax-general/title-10/subtitle-7/section-10-751/) states that the md_ctc is reduced by the federal ctc (basically making the whole credit refundable, however in [2023](https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-751-tax-credit-for-qualified-child) this paragraph was revoked and the md_ctc amount is not reduced and only the excess is refundable. I have modeled the 2023 tax rules, is there a way to implement both based on the year?

Reply:
Could you use formula_2021 and formula_2023?

Reply:
used, let me know if this structure works

Reply:
not separate variables, separate formulas

see https://openfisca.org/doc/coding-the-legislation/40_legislation_evolutions.html#formula-evolution

Reply:
Should work now.